### DISCUSSION TITLE
我用这个项目申请了一个 JetBrains 全家桶的许可

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#702] [Ideas]
结果发现问题好多

![图片](https://user-images.githubusercontent.com/13673702/157452303-b8459581-a4de-4235-8f57-507e69bbf7d5.png)

**Comment by mark9804** [comment]
呃，我之前想跟老大哥说我们项目可以嫖全家桶的，后来想想老大哥 vim 用的飞起，我又是学生可以让他免费送，就没说了，我是从大学开始一直在用全家桶，那时候他们还没中文

HTML 主要是因为我直接写的 `.js` 不是 `.vue` ，所以有识别问题，CSS 确实是上一个 commit inherit 没加全 4fde454b6dc14a7f5578ee20704a68eff8114c57 ，可以配置忽略掉一些检查选项

<details>

<img width="626" alt="image" src="https://user-images.githubusercontent.com/9006264/157454712-56bd8547-638f-45d7-bb1b-3fb894dc776e.png">
</details>

**Reply by Arondight** [reply]
编辑器还是太局限了，代码的水平等同于使用者的水平， ide 可以提升一下

**Reply by mark9804** [reply]
<img width="245" alt="image" src="https://user-images.githubusercontent.com/9006264/157473019-bc07dae5-67d2-4cb2-bf77-1e2d123c7486.png">

**Comment by mark9804** [comment]
<img width="640" alt="image" src="https://user-images.githubusercontent.com/9006264/165676972-4fe79772-e61d-45a6-8e42-c7fcfd2eb434.png">

新启动界面好几把酷炫

**Reply by Arondight** [reply]
感觉变丑了

**Reply by mark9804** [reply]
这个版本很容易崩……刚刚写文档的时候崩掉了一次，本地 VCS log 也没有我这一个小时的记录，还好只是文档
